src/Nri/Ui/TextInput/V5.elm

```diff
@@ -110,39 +121,48 @@ view_ theme model =
 
         (InputType inputType) =
             model.type_
+
+        maybeStep =
+            if inputType.fieldType == "number" then
+                [ step "any" ]
```

Comment:
"any" in the spinner control looks like it might default to "1", same as not setting it. Why do we need to? Alternatively, why aren't we allowing control of the value here?

Reply:
What's the spinner control? The presence of the "step" property is what allows for input of a float value without the text input becoming invalid.  "any" seemed to not mess up ints, not error on floats, and be good enough. We could allow control of the value! But didn't have a current need--not sure how we like to balance future proofing vs doing the least invasive change.

Reply:
spinner control is the arrow keys working to increase/decrease the value and the arrows here:

<img width="190" alt="image" src="https://user-images.githubusercontent.com/355401/69181843-7d9d6400-0ad5-11ea-8bb1-bd20497ff525.png">

if that's not the goal here this seems fine!

Reply:
Oh cool! New terminology for me 👏

Reply:
TIL also!